documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -3,77 +3,72 @@
-You have to first downgrade through one of the Strimzi versions in-between, disable the `UseStrimziPodSets` feature gate, and then downgrade to Strimzi 0.27 or earlier.
+IMPORTANT: With the `UseStrimziPodSets` feature gate permanently enabled, direct downgrades from Strimzi 0.35 and newer to Strimzi 0.27 or earlier are not possible. You must first downgrade through one of the Strimzi versions in-between, disable the `UseStrimziPodSets` feature gate, and then downgrade to Strimzi 0.27 or earlier.
 
 [id='ref-operator-stable-connect-identities-feature-gate-{context}']
-== StableConnectIdentities feature gate
+=== StableConnectIdentities feature gate
+
+The `StableConnectIdentities` feature gate introduced the `StrimziPodSet` custom resource for managing Kafka Connect and Kafka MirrorMaker 2 pods, replacing the use of Kubernetes `Deployment` resources.
+
+`StrimziPodSet` resources give the pods stable names and stable addresses, which do not change during rolling upgrades, replacing the use of Kubernetes `Deployment` resources.
+
+IMPORTANT: With the `StableConnectIdentities` feature gate permanently enabled, direct downgrades from Strimzi 0.39 and newer to Strimzi 0.33 or earlier are not possible.
+You must first downgrade through one of the Strimzi versions in-between, disable the `StableConnectIdentities` feature gate, and then downgrade to Strimzi 0.33 or earlier.
 
-The `StableConnectIdentities` feature gate has moved to GA, which means it is now permanently enabled and cannot be disabled.
-The `StrimziPodSet` resources are now always used to manage Kafka Connect and Kafka MirrorMaker 2 pods instead of using Kubernetes `Deployment` resources.
-`StrimziPodSet` resources give the pods stable names and stable addresses, which do not change during rolling upgrades.
-This helps to minimize the number of rebalances of connector tasks.
+== Preview feature gates (alpha/beta)
 
-IMPORTANT: With the `StableConnectIdentities` feature gate permanently enabled, it is no longer possible to downgrade directly from Strimzi 0.39 and newer to Strimzi 0.33 or earlier.
-You have to first downgrade through one of the Strimzi versions in-between, disable the `StableConnectIdentities` feature gate, and then downgrade to Strimzi 0.33 or earlier.
+Preview feature gates that have not yet reached the GA stage can be enabled or disabled. 
+A preview feature gate provides an opportunity for assessment before its functionality is permanently incorporated into AMQ Streams.
 
 [id='ref-operator-use-kraft-feature-gate-{context}']
-== (Preview) UseKRaft feature gate
+=== UseKRaft feature gate
 
 The `UseKRaft` feature gate has a default state of _disabled_.
 
-The `UseKRaft` feature gate deploys the Kafka cluster in the KRaft (Kafka Raft metadata) mode without ZooKeeper.
+The `UseKRaft` feature gate deploys a Kafka cluster in KRaft (Kafka Raft metadata) mode without ZooKeeper.
```

Comment:
```suggestion
The `UseKRaft` feature gate deploys a Kafka cluster in KRaft mode without ZooKeeper.
KRaft is a term that refers to the Kafka Raft dialect used for replicating the cluster metadata.
```

Reply:
I would not change this. I think the original is more understandable to users and makes more sense in general. I do not think _Raft dialect_ has any real meaning.

Reply:
It's shorter, but not more understandable IMO. The term "Raft dialect" was mentioned in the [original KIP](https://cwiki.apache.org/confluence/display/KAFKA/KIP-595%3A+A+Raft+Protocol+for+the+Metadata+Quorum).

Reply:
Just because it is used in some KIP does not mean it is understandable. If nothing else, the audiences are very different.

Reply:
Raft dialect isn't used in the Kafka docs. I don't think we should start using it.